.github/workflows/validate-workflows.yaml

```diff
@@ -0,0 +1,49 @@
+name: Validate Github Workflows
+
+on:
+  pull_request: {}
```

Comment:
Commenting here to make sure we add [`on.pull_request.paths`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths) to ensure we only run when changes are made to workflows, as discussed on Slack.